common-model/src/main/java/com/sequenceiq/common/api/command/RemoteCommandsExecutionRequest.java

```diff
@@ -0,0 +1,36 @@
+package com.sequenceiq.common.api.command;
+
+import java.util.Set;
+
+public class RemoteCommandsExecutionRequest {
```

Comment:
There is no ApiModelProperty and validator annotations on the request object. Could you please add them?